src/components/Conversion/ConversionNOB.vue

```diff
@@ -49,14 +49,14 @@
         <div v-if="!onEditMode" class="d-flex justify-space-between align-start">
           <span id="naics-summary">{{ naicsSummary }}</span>
 
-          <div v-if="!hasNaicsChanged" class="mt-n2 mr-n3">
+          <div v-if="!hasNaicsChanged" class="my-n2 mr-n3">
```

Comment:
This is to effectively make the action button shorter so the whitespace below this line is the same as above it.

Before:
![image](https://user-images.githubusercontent.com/10002889/196290781-e3fa4522-01ca-4661-8099-339fc8d9e627.png)

After:
![image](https://user-images.githubusercontent.com/10002889/196290824-a0f20892-db7a-4b01-912f-dcf6316d3840.png)